06-building-a-program-start-to-finish/solving-the-problem/output/app.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @class InvertedIndex
+ */
+class InvertedIndex {
+  /**
+   * Class Constructor
+   * @constructor
+   */
+  constructor() {
+    this.allIndices = {};
+  }
+  /**
+   * Reads the data from the file being uploaded
+   * @param  {File} file - Uploaded file to be read.
+   * @return {void}
+   */
+  readFile(file) {
+    this.file = file;
```

Comment:
Initially I didn't want to make any method of the class static. This is because I want to be able to call all the methods through an instance of the class. 
Nevertheless, this has been refactored to make it those methods static.